### ISSUE TITLE
Issue with decimals

### ISSUE THREAD

**Issue by MechEng70** [shbatm/hacs-isy994#92]
Loaded with the HACS repository.  On 4.0.3.

Noticed that the temperature, humidity and luminance is off by one decimal.

![Screenshot_20230203-121045_Home_Assistant](https://user-images.githubusercontent.com/11697710/216703227-df568c7d-e856-4846-9124-ae112620b385.png)
![Screenshot_20230203-121055_Home_Assistant](https://user-images.githubusercontent.com/11697710/216703232-563ce11a-ac0b-454e-9b60-f7d9afb25027.png)

**Comment by shbatm** [comment]
Are you using comma or decimal for separation? Looks like more than 1 decimal ... Unless your sensor is normally 386F

**Comment by shbatm** [comment]
Try commenting out this line:

https://github.com/shbatm/hacs-isy994/blob/c8884a35825605e1639443d7617653d2d14e8201/custom_components/isy994/sensor.py#L200

If that doesn't work try changing the `0` to `self.target.precision`

**Comment by MechEng70** [comment]
I am using decimal.  I have feeling that it is converting from C to F and then the decimal issue.  Should be 2 decimal places to the left.  
Otherwise it would be 38.6 + 32 = 70.6F which makes more sense.... 

commented out line 200 and restarted HA.  no change to the decimal placement.
Where is self.target.precision?  Did a search and couldn't find it.

**Comment by shbatm** [comment]
Make the line above:

```python
value = convert_isy_value_to_hass(value, uom, self.target.precision)
```

**Comment by MechEng70** [comment]
Apologies, I am really tired today!  I did the change.  
Reloaded the integration and it didn't work.  Restarted HA and it did work.  For the record, I was off by 0.3F.  Temp is displaying 70.322F.  Much more comfortable than 386F

**Comment by shbatm** [comment]
Which ended up working, commenting out or adding back in the precision?

And ya, any changes to the custom components code requires a full restart of HA.

And no worries , on mobile otherwise I'd be able to dig a little more myself.

**Comment by MechEng70** [comment]
Replacing the 0 with self.target.precision

**Comment by shbatm** [comment]
Closed in 4.0.4
